### PR TITLE
Introduce Aspid to Harmony

### PR DESCRIPTION
**Pull request by Golinth**
## About the PR

Added a previously derotated wizden map: Aspid, while reworking some of the more contentious departments. I am absolutely looking for feedback and areas to improve the map, so please do not hesitate to leave comments. 

**Primary Changes:**

- Restructured Med
- Restructured Sci
- Restructured Salv, moving it south of Cargo, replacing the derelict Docking Arm.
- Reworked dorms, replacing the old EVA storage
- Moved EVA Storage to the old park area, directly north of the HoP 
- Added AI to the previous Salv area.
- Added Tesla to Engineering, reduced AME to 1 core.
- Added named cameras to anywhere missing AI LoS.
- Removed round-start cryo spawn.
- Reduced the amount of Powergamer loot
- Added a brigmed area to Sec.
- Added roundstart Ripley to cargo
- Made brig lockdown button functional
- Made bridge lockdown button functional
- Upped max playercount to 65 and min to 30.

## Why / Balance

Spanky requested that I move this downstream, as Harmony was looking to return Aspid, and I had already performed some work to modernize the map to current Wizden standards.

## Media

Full map:
Aspid-0

Medbay rework: 
image

Science rework: 
image

AI Core:
image

Salvage: 
image

Tesla gen: 
image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl:
- add: Added Aspid, a medium pop map based around a luxury star cruiser!

### CODE REVIEW

**Review by spanky-spanky — CHANGES_REQUESTED** [review]
Please move the map and map prototype files into their respective _Harmony folders at:
`/Resources/Maps/_Harmony`
`/Resources/Prototypes/_Harmony/Maps`
And move - Aspid in /Pools/default.yml to be under # Harmony. Otherwise looks good!

**Comment by spanky-spanky** [comment]
Also small note, but upstream instructions were given to unmap intellicards now they they will spawn in the RD's locker. You can include it in this PR or save it for future.

**Comment by Golinth** [comment]
I'll save it for the next time I update the map from player reviews, or after Emisse reviews. Whichever comes first. I'll go ahead and write it down though.

File: Resources/Prototypes/_Harmony/Maps/aspid.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+- type: gameMap
+  id: Aspid
+  mapName: 'NCS Aspid'
+  mapPath: /Maps/aspid.yml
```
[the remark this answers is not in the document]
Comment (Golinth) [reply Resources/Prototypes/_Harmony/Maps/aspid.yml]: Oops. Fixed

**Review by spanky-spanky — APPROVED** [review]

**Review by spanky-spanky — APPROVED** [review]

**Pull request merged**